src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -1,4 +1,4 @@
-# Intracranial Electroencephalography
+# intracranial Electroencephalography 
```

Comment:
one more unnecessary space at the end of the line: And Why did you turn the upper case i into a smaller case i? I think it should stay upper case, like in the TOC: https://github.com/bids-standard/bids-specification/blob/ece9e8737d15a4ca9e5453dc8c473abbd3e4dd91/mkdocs.yml#L30

Reply:
when I added the pdf generation files after syncing my fork with the original repo, there were some changes in the markdown files that I had to fix manually so these were oversights. I've fixed them now.